src/dependency-manager/src/service-config/v1.ts

```diff
@@ -209,8 +199,19 @@ class InterfaceSpecV1 extends BaseSpec {
   @IsString({ always: true })
   host?: string;
 
+  @Type(() => Number)
```

Comment:
I think you can remove. Now that we have validation this shouldn't be an issue except in our own tests.

Reply:
Are you sure? Both the tests and local deployments had problems if I didn't include this

Reply:
Yeah that's because the tests themselves defined as strings which would fail validation if we ran it. Try removing now that you updated the tests.

Reply:
Yeah you're right. Updated